python/tk_desktop2/websockets/requests/sgc_actions/open_version_draft.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2018 Autodesk, Inc.  All rights reserved.
+#
+# Use of this software is subject to the terms of the Autodesk license agreement
+# provided at the time of installation or download, or which otherwise accompanies
+# this software in either electronic or hard copy form.
+#
+
+import sgtk
+from ..request import WebsocketsRequest
+from ....shotgun_entity_path import ShotgunEntityPath
+
+logger = sgtk.LogManager.get_logger(__name__)
+
+
+class OpenVersionDraftInSGCreateWebsocketsRequest(WebsocketsRequest):
+    """
+    Requests that sets the current draft in the Shotgun Create player.
+
+    This is a 'fire-and-forget' command that doesn't return anything.
+
+    Parameters dictionary is expected to be of the following form:
+
+    {
+        'task_id': 123,
+        'path': "/path/to/a/file/to/load/as/draft",
+        ['version_data': {"sg_field_1": "sg_field_1_value", "sg_field_2", "sg_field_2_value"}]
+    }
+
+    - task_id is required and needs to point to a task associated with an entity.
+    - path is required and needs to point to a valid path on disk.
+    """
+
+    def __init__(self, connection, id, parameters):
+        """
+        :param connection: Associated :class:`WebsocketsConnection`.
+        :param int id: Id for this request.
+        :param dict params: Parameters payload from websockets.
+        """
+        super(OpenVersionDraftInSGCreateWebsocketsRequest,
+              self).__init__(connection, id)
+
+        self._bundle = sgtk.platform.current_bundle()
+
+        if "task_id" not in parameters:
+            raise ValueError(
+                "%s: Missing required 'task_id' key "
+                "in parameter payload %s" % (self, parameters)
+            )
+        self._task_id = parameters["task_id"]
+
+        if "path" not in parameters:
+            raise ValueError(
+                "%s: Missing required 'path' key "
+                "in parameter payload %s" % (self, parameters)
+            )
+        self._draft_path = parameters["path"]
+
+        self._version_data = parameters.get("version_data", "{}")
+
+    @property
+    def analytics_command_name(self):
+        """
+        The command name to pass to analytics.
+        """
+        return "open_version_draft"
+
+    def execute(self):
+        """
+        Execute the payload of the command
+        """
+        try:
+            engine = sgtk.platform.current_bundle()
+
+            # open task - resolve link and project
+            task_data = engine.shotgun.find_one(
+                "Task",
+                [["id", "is", self._task_id]],
+                ["project", "entity"]
+            )
+
+            if task_data is None:
+                raise ValueError(
+                    "Task id %d cannot be found in Shotgun!" % (self._task_id,))
```

Comment:
I'm nitpicking, but if you do the following when you're line breaking calls it'll look like the rest of our codebase:

```
raise ValueError(
    "Error message..."
)
```

Reply:
If you want to avoid these nit-picky questions in the future, reach out to me. We're going to be adhering to the `black` coding standard and all of this formatting will be automated for us. I'll help you set it up.